python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -364,6 +368,24 @@ def __infer_offsets(
             }
         )
 
+    def __dask_array_from_numpy(
+        self, array: np.ndarray, client: distributed.client.Client, npartitions: int
+    ):
+        split_array = np.array_split(array, npartitions)
+        shapes = [n.shape for n in split_array]
+        scattered_array = client.scatter(split_array)
+        del split_array
+
+        dask_array = dar.concatenate(
+            [
+                dar.from_delayed(s, shape=shapes[i], dtype=array.dtype)
+                for i, s in enumerate(scattered_array)
+            ]
+        )
+        del scattered_array
+
```

Comment:
We should use `da.from_array(x)  ` here. 

https://docs.dask.org/en/stable/generated/dask.array.from_array.html

Reply:
fixed